message/utils/checkIfEmpty.js

```diff
@@ -1,4 +1,6 @@
-const checkIfEmpty = (str) => {
+const checkIfEmpty = (input) => {
```

Comment:
I think a better way verify that the string is empty would be the follow
```javascript
return (!input || input.length === 0 );
```

Reply:
I tried a similar way, but the problem i that a empty string is considered as a falsy, so the checkIfEmpty util seems a better way to me, maybe?

Reply:
I mean, it checks if the string or the number is empty, not if it is false